cmd/juju/model/defaultscommand.go

```diff
@@ -2,18 +2,31 @@
 // Licensed under the AGPLv3, see LICENCE file for details.
 package model
 
+// tests to add
+// Test for no arg
+// Test for region only
+// Test for cloud only - fail because unknown key -- just cloud is invalid
+// Test for cloud/region - valid
+// TEst for cloud/region - invalid
+
 import (
 	"bytes"
 	"io"
 	"sort"
 	"strings"
 
+	names "gopkg.in/juju/names.v2"
+
 	"github.com/juju/cmd"
 	"github.com/juju/errors"
 	"github.com/juju/gnuflag"
 	"github.com/juju/utils/keyvalues"
 
+	"github.com/juju/juju/api"
+	"github.com/juju/juju/api/base"
+	cloudapi "github.com/juju/juju/api/cloud"
 	"github.com/juju/juju/api/modelmanager"
+	jujucloud "github.com/juju/juju/cloud"
```

Comment:
I recall looking for this and followed the style in cmd/juju/controller/addmodel.go